src/index.html

```diff
@@ -5,10 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Flexbox</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="logo" href="#">
+        <img class="main images" src="images/logo.png"
+        alt="Company Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link">Apple</a>
+          </li>
```

Comment:
Let's make links more informative
```suggestion
          <li class="nav__item">
            <a href="#apple" class="nav__link">Apple</a>
          </li>
```

Reply:
not fixed
https://github.com/mate-academy/layout_moyo-header/pull/3341/files#r1227995244